### PR TITLE
New: Automation Integrations introducing send email, send slack, and ruleset alert

### PR DESCRIPTION
**Pull request by ericrpowers**
#### Description
The next phase of automation support is Integrations (which are what we call actions). To make it a better UX, we want to covert relevant actions into recipes:
- Send Slack message
- Send Email(s)

We also want to use recipes for introducing our structure automations to customers. The first being Ruleset Alert which will do the following:
- Generate an alert for new flows matching a specific ruleset in a list of rulesets.
- This may indicate an attempt to violate a specify guardrail or a more sophisticated attack.
- The user must be able to specify what rulesets they wants to generate alerts upon (including the Namespace Default Ruleset).

TODO:
- [x] Integrate Severity, Title, and Content into Ruleset Alert (https://redlock.atlassian.net/wiki/spaces/RED/pages/3389128900/Automations+Alerts)
- [x] JS work to fill in condition for Ruleset Alert
- [x] Flow Details

> Fixes: https://redlock.atlassian.net/browse/CNS-3222

### CODE REVIEW

File: automations/recipe-automation-action-send-email.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+APIVersion: 1
+label: recipe:automation-action:send-email
+data:
+  recipes:
+    - name: Create a Send Email Automation Action
+      description: Creates an automation action that creates an alerm and sends an email to the specified recipient(s).
```
Comment (npylai) [inline automations/recipe-automation-action-send-email.yaml:6]: "alarm"

File: automations/recipe-automation-ruleset-alert.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+APIVersion: 1
+label: recipe:automation:ruleset-alert
+data:
+  recipes:
+    - name: Create a Ruleset Alert Automation
+      description: Creates an automation that will raise an alert when a ruleset reaches its limit in matcing flow report.
```
Comment (npylai) [inline automations/recipe-automation-ruleset-alert.yaml:6]: "matching"

File: automations/automation-condition-flow-reports-matching-rulesets.yaml [hunk 3]
```diff
@@ -0,0 +1,111 @@
+APIVersion: 1
+label: automation:condition:flows_reports_matching_rulesets
```
Comment (npylai) [inline automations/automation-condition-flow-reports-matching-rulesets.yaml:2]: should it be "flow reports"?

File: automations/automation-condition-flow-reports-matching-rulesets.yaml [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+identities:
+- automationcondition
+data:
+  automationconditions:
+  - key: '@condition:flows_reports_matching_rulesets'
```
Comment (npylai) [inline automations/automation-condition-flow-reports-matching-rulesets.yaml:7]: same as above "flow_reports..." vs "flows_reports..."?

**Comment by ericrpowers** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "automation-integration",
    "component": "backend",
    "pr-id": "1471",
    "commit-sha": "567d76cb4d5aa3518d8df83437268d35433ded5f",
    "pipeline": "master"
  },
  {
    "project": "automation-integration",
    "component": "cookbook",
    "pr-id": "40",
    "commit-sha": "be94a1b655114a4bc0c1685b55abe82cecd3adf1",
    "pipeline": "master"
  },
  {
    "project": "automation-integration",
    "component": "gaia",
    "pr-id": "746",
    "commit-sha": "1369613410c51217150b66d5057b9ee995a1118d",
    "pipeline": "master"
  }
]
```

**Pull request merged**